Source/Dafny/Scanner.cs

```diff
@@ -274,49 +274,49 @@ public class Scanner {
 		for (int i = 95; i <= 95; ++i) start[i] = 1;
 		for (int i = 99; i <= 122; ++i) start[i] = 1;
 		for (int i = 49; i <= 57; ++i) start[i] = 55;
-		start[97] = 56; 
-		start[98] = 57; 
-		start[39] = 58; 
-		start[48] = 59; 
-		start[34] = 27; 
-		start[64] = 105; 
-		start[58] = 60; 
-		start[44] = 34; 
-		start[124] = 106; 
-		start[8226] = 38; 
-		start[46] = 107; 
-		start[96] = 39; 
-		start[59] = 40; 
-		start[61] = 108; 
-		start[123] = 42; 
-		start[125] = 43; 
-		start[91] = 44; 
-		start[93] = 45; 
-		start[40] = 46; 
-		start[41] = 47; 
-		start[60] = 109; 
-		start[62] = 110; 
-		start[33] = 111; 
-		start[8800] = 49; 
-		start[42] = 50; 
-		start[43] = 80; 
-		start[45] = 112; 
-		start[126] = 81; 
-		start[35] = 86; 
-		start[8804] = 88; 
-		start[8805] = 89; 
-		start[8660] = 91; 
-		start[8658] = 93; 
-		start[8656] = 94; 
-		start[38] = 113; 
-		start[8743] = 96; 
-		start[8744] = 98; 
-		start[172] = 99; 
-		start[8704] = 100; 
-		start[8707] = 101; 
-		start[47] = 102; 
-		start[37] = 103; 
-		start[94] = 104; 
+		start[97] = 56;
+		start[98] = 57;
+		start[39] = 58;
+		start[48] = 59;
+		start[34] = 27;
+		start[64] = 105;
+		start[58] = 60;
+		start[44] = 34;
+		start[124] = 106;
+		start[8226] = 38;
+		start[46] = 107;
+		start[96] = 39;
+		start[59] = 40;
+		start[61] = 108;
+		start[123] = 42;
+		start[125] = 43;
+		start[91] = 44;
+		start[93] = 45;
+		start[40] = 46;
+		start[41] = 47;
+		start[60] = 109;
+		start[62] = 110;
+		start[33] = 111;
+		start[8800] = 49;
+		start[42] = 50;
+		start[43] = 80;
+		start[45] = 112;
+		start[126] = 81;
+		start[35] = 86;
+		start[8804] = 88;
+		start[8805] = 89;
+		start[8660] = 91;
+		start[8658] = 93;
+		start[8656] = 94;
+		start[38] = 113;
+		start[8743] = 96;
+		start[8744] = 98;
+		start[172] = 99;
+		start[8704] = 100;
+		start[8707] = 101;
+		start[47] = 102;
+		start[37] = 103;
+		start[94] = 104;
```

Comment:
The file Scanner.cs (and Parser.cs) are generated by the Coco parser generator. Let's not change the whitespace in these files, since that will be overridden next time we run Coco. (Plus, since these files are not hand edited, the annoyance of trailing whitespace is not high.)

Reply:
Done.